Libplanet.Explorer/Queries/TransactionQuery.cs

```diff
@@ -29,31 +29,23 @@ public TransactionQuery(IBlockChainContext<T> context)
             _context = context ?? throw new ArgumentNullException(nameof(context));
 
             Field<NonNullGraphType<ListGraphType<NonNullGraphType<TransactionType<T>>>>>(
-                "transactions",
-                arguments: new QueryArguments(
-                    new QueryArgument<AddressType>
-                    {
-                        Name = "signer",
-                        DefaultValue = null,
-                    },
-                    new QueryArgument<AddressType>
-                    {
-                        Name = "involvedAddress",
-                        DefaultValue = null,
-                    },
-                    new QueryArgument<BooleanGraphType>
-                    {
-                        Name = "desc",
-                        DefaultValue = false,
-                    },
-                    new QueryArgument<IntGraphType>
-                    {
-                        Name = "offset",
-                        DefaultValue = 0,
-                    },
-                    new QueryArgument<IntGraphType> { Name = "limit" }
-                ),
-                resolve: context =>
+                "transactions")
+                .Argument<Address>(
+                    "signer",
+                    true,
+                    "Filter by signer (if given).",
+                    arg => arg.DefaultValue = null)
+                .Argument<Address>(
+                    "involvedAddress",
+                    true,
+                    "Filter by involved addresses (if given).",
+                    arg => arg.DefaultValue = null)
+                .Argument<NonNullGraphType<BooleanGraphType>>(
```

Comment:
It was initially an optional argument.

```suggestion
                .Argument<BooleanGraphType>(
```

Reply:
This was option, but is it any more good to make it optional and give it `false` as its default value?  How does differ `null` from `false` here?